src/FeedInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace BazaarVoice;
+
+use BazaarVoice\Elements\FeedElementInterface;
+
+interface FeedInterface
```

Comment:
this interface is really necessary? I noticed that everyone who implements this contract also extend from `AbstractFeed`

the methods in this contract can all be moved to `AbstractFeed`, changing its signature to `abstract` to force an implementation

Reply:
this interface is good in case you need to customize the Feed implementation outside this package scope without forcing to extend a given class.

It's not a problem rely on an interface, specially in a opensource package.

Reply:
It is, when you're making premature abstractions, thinking in the future.

Anyway, i already approved, was just a comment.
This does't make any difference.